### PR TITLE
Release 1.2.2

### PR DESCRIPTION
**Pull request by KochTobi**
1.2.2 (2021-03-25)
---------------------------

**Added**

* Add `maven enforcer plugin <https://maven.apache.org/enforcer/maven-enforcer-plugin>`_ to show dependency conflicts

**Fixed**

**Dependencies**

* Bump ``data-model-lib:2.0.0`` -> ``2.4.0`` (`#8 <https://github.com/qbicsoftware/sample-status-updater-cli/pull/8>`_)

**Deprecated**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -62,12 +62,37 @@
...
+								<dependencyConvergence/>
+								<banDuplicatePomDependencyVersions/>
+							</rules>
+							<fail>false</fail>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
```
Comment (wow-such-code) [inline pom.xml:97]: What does this do? Should the change be described in the change log?

Comment (KochTobi) [reply pom.xml:97]: good point

Comment (wow-such-code) [reply pom.xml:97]: ah, guess it's this: "Uses maven enforcer plugin to show depencency conflicts"
if you add this information to squash and merge I'm happy!

**Review by wow-such-code — APPROVED** [review]
not sure if the plugin changes need to be added to the changelog, but please mention them in the merge

File: pom.xml (hunk 1 above)
Comment (KochTobi) [reply pom.xml:97]: @sven1103 I do not know what this actually does neither why it was added. Can you remember?

**Review by wow-such-code — APPROVED** [review]
great!

**Pull request merged**